plugins/inputs/nfs/nfs.go

```diff
@@ -0,0 +1,344 @@
+package nfs
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFS struct {
+	Iostat   bool
+	Fullstat bool
+}
+
+var sampleConfig = `
+  ## Read fewer metrics (iostat)
```

Comment:
you went a bit _too_ brief, you can go multiple lines, but summarize the metrics it will collect, you can give a few examples if you need to